apps/judicial-system/web/src/routes/Court/Indictments/Completed/Completed.tsx

```diff
@@ -154,6 +162,43 @@ const Completed: FC = () => {
   const hasMergeCases =
     workingCase.mergedCases && workingCase.mergedCases.length > 0
 
+  const marginSpaceBetweenButtons = 2
+
+  const defendantCheckboxes = [
+    {
+      label: 'Leiðbeiningar um endurupptöku útivistarmála',
+      value: 'instructionsOnReopeningOutOfCourtCases',
```

Comment:
Tha values here can come from the GraphQL enum generated as described earlier. The enum should be imported from the generated schema file.